src/spetlr/delta/delta_handle.py

```diff
@@ -25,22 +28,49 @@ class DeltaHandleInvalidFormat(DeltaHandleException):
 
 
 class DeltaHandle(TableHandle):
-    def __init__(self, name: str, location: str = None, data_format: str = "delta"):
+    def __init__(
+        self,
+        name: str,
+        location: str = None,
+        data_format: str = "delta",
+        options_dict: dict = None,
+        ignore_changes: bool = True,
+        stream_start: datetime = None,
+        max_bytes_per_trigger: int = None,
+    ):
+        """ """
         self._name = name
         self._location = location
         self._data_format = data_format
 
         self._partitioning: Optional[List[str]] = None
-
         self._validate()
 
+        self._options_dict = (
+            {} if options_dict is None or options_dict == "" else options_dict
+        )
+
+        self._options_dict.update({"ignoreChanges": str(ignore_changes)})
+
+        if stream_start and stream_start != "":
+            self._options_dict["startingTimestamp"] = stream_start.strftime(
+                "%Y-%m-%dT%H:%M:%S.%fZ"
+            )
+
+        if max_bytes_per_trigger and max_bytes_per_trigger != "":
+            self._options_dict["maxBytesPerTrigger"] = max_bytes_per_trigger
+
     @classmethod
     def from_tc(cls, id: str) -> "DeltaHandle":
         tc = Configurator()
         return cls(
             name=tc.table_property(id, "name", ""),
             location=tc.table_property(id, "path", ""),
             data_format=tc.table_property(id, "format", "delta"),
+            options_dict=tc.table_property(id, "options_dict", ""),
```

Comment:
here you read everything in as string, that is not in accordance with your types in the constructor.

Reply:
removed it from tc